openwisp_radius/base/models.py

```diff
@@ -1055,15 +1070,38 @@ def clean(self):
                     )
                 }
             )
+        # Clean `freeradius_allowed_hosts`
+        iplist = self.freeradius_allowed_hosts_list
```

Comment:
I'd change this var to `ip_list` for consistency with `ip_addr` below (which you could rename `ip_address` since it's not very long and help readability, both suggested changes improve readability.